src/me/components/Resources.tsx

```diff
@@ -64,6 +69,9 @@ const ResourceLists: FC = () => {
           </Panel>
         </>
       )}
+      {isFlagEnabled('paygCheckoutCredit') && paygCreditEnabled && (
```

Comment:
Not sure if this is relevant or not, but we might need to make this cloud only, with a check for uiUnification since this’ll be weird in OSS and tools

Reply:
don't think this is relevant as the payg credit start date won't be available and thus this should never be visible.

Reply:
It's relevant. In order to determine the `paygCreditEnabled` value we make a request to quartz for the `paygCreditStartDate` right? `new Date(undefined) === Invalid Date`. What I'm getting at is that we need to make sure that we only derive that data for Cloud users that aren't tools.